app/src/main/java/org/lineageos/twelve/fragments/PlaylistsFragment.kt

```diff
@@ -120,6 +159,32 @@ class PlaylistsFragment : Fragment(R.layout.fragment_playlists) {
         }
     }
 
+    private fun openCreateNewPlaylistDialog() {
+        val context = requireContext()
+
+        lateinit var editText: EditText
+
+        MaterialAlertDialogBuilder(context)
```

Comment:
Remember to add a style for rounded corner

Reply:
![Screenshot_20240922-123106](https://github.com/user-attachments/assets/54d5e7a6-0c79-4703-a658-d46e5a60f004)
Looks like it only applies to PreferenceFragment